### ISSUE TITLE
Include surge foils in image downloads?

### ISSUE THREAD

**Issue by PurpleCrowbar** [magefree/mage#12096]
Scryfall contains all variants of a card's appearance, including new artworks / borders, etc. It also contains "surge foils", a printing of the card that can only be foil, which specifically appears in collector products. Shown below are a couple of examples of the original card image and the surge foil card image on Scryfall:
<details><summary>Piper Wright</summary>

![image](https://github.com/magefree/mage/assets/26198472/58170ed8-7b2b-4627-b98f-f041af711e1d)
![image](https://github.com/magefree/mage/assets/26198472/ef5a7fa2-fba5-449a-a156-328ad1fa4773)
</details>
<details><summary>Pip-Boy 3000</summary>

![image](https://github.com/magefree/mage/assets/26198472/9a941516-1c7d-4ae5-939a-63296450333c)
![image](https://github.com/magefree/mage/assets/26198472/eb39d119-3cff-4546-b472-927e043f0dcd)
</details>

If you're having a difficult time differentiating between them, it's because the images are 1:1 identical except for the number in the bottom-left corner. In the case of both these cards, they also have extended art variants, which in turn have surge foil variants, despite the fact that the image is, again, the same (see [here](https://scryfall.com/search?as=grid&order=released&q=%21%22Piper+Wright%2C+Publick+Reporter%22+include%3Aextras&unique=prints) and [here](https://scryfall.com/search?as=grid&order=released&q=%21%22Pip-Boy+3000%22+include%3Aextras&unique=prints)). Given the HD quality of Scryfall images, it seems spectacularly inefficient to have users download the exact same HD image twice for each variant of these cards, and it additionally further serves to clog up the deck editor when searching for image variants for cards. Would it not be more sensible to exclude surge foils from the image downloads?

**Comment by JayDi85** [comment]
Xmage uses scryfall sets structure. So if scryfall set contains that cards then it must be included in xmage too.

**Comment by Susucre** [comment]
> Xmage uses scryfall sets structure. So if scryfall set contains that cards then it must be included in xmage too.
While I do agree with that, we could have a lighter image mode that downloads at most 1 image per card name, across all sets and variations (the oldest? the newest? the one that scryfall uses by default through its API?).
Every set now has 4/5 version of a lot of cards, and it really does take a lot of disk space.

**Comment by JayDi85** [comment]
Yes, it can be part of deck editor images replacement (#5472) and client side images replacement (#7182).
